server.js

```diff
@@ -74,12 +81,19 @@
     }
 
     var knownTilesetFormats = [/\.b3dm/, /\.pnts/, /\.i3dm/, /\.cmpt/, /\.glb/, /tileset.*\.json$/];
-    app.get(knownTilesetFormats, checkGzipAndNext);
-
-    app.use(function(req, res, next) {
-        res.header("Access-Control-Allow-Origin", "*");
-        res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
-        next();
+    app.use(knownTilesetFormats, checkGzipAndNext);
+
+    // Custom code for serving TilesetWithExpiration. When points.pnts is requested it cycles between the tiles in the cache folder.
+    var expirationPntsPath = '/tilesets/TilesetWithExpiration/points.pnts';
+    var expirationCacheDirectory = '/tilesets/TilesetWithExpiration/cache/';
+    var expirationCacheLength = 5;
+    var expireCount = 0;
+
+    app.use(expirationPntsPath, function(req, res) {
+        var pntsPath = expirationCacheDirectory +  'points_' + expireCount + '.pnts';
+        expireCount = (expireCount + 1) % expirationCacheLength;
+        res.sendFile(pntsPath, {root: __dirname});
+        // Don't call next() because we don't need to run the express.static middleware
```

Comment:
I think the more correct thing to do is use `app.get` here and always call next.  Similarly, `express.static` should be an `app.get(/.*/, express.static(__dirname));)`.

`app.get` is applied "waterfall" style, so the earliest `app.get` that matches a url wins and then processing ends.
`app.use` is run on everything, which clearly is not what we want for these requests.

If you try these changes and run into problems, we can just punt for now and I'll write an issue to revisit.